qtox/build_utils.sh

```diff
@@ -109,5 +119,33 @@ parse_arch() {
     usage
     exit 1
   fi
+
+  if [ "$BUILD_TYPE" = "release" ]; then
+    CMAKE_BUILD_TYPE="Release"
+  elif [ "$BUILD_TYPE" = "debug" ]; then
+    CMAKE_BUILD_TYPE="RelWithDebInfo"
```

Comment:
hmmm, debug?

Reply:
No, that's way too slow. Release with debug info is ok for dependencies.

Reply:
Ah, for deps. I always mod my Debug profile with -Og .